src/pages/views/components/reports/reportSummary/reportSummaryItems.test.tsx

```diff
@@ -1,30 +1,44 @@
-import { shallow } from 'enzyme';
+import { render, screen } from '@testing-library/react';
 import React from 'react';
+import { FetchStatus } from 'store/common';
 import * as utils from 'utils/computedReport/getComputedReportItems';
 
 import { ReportSummaryItems, ReportSummaryItemsProps } from './reportSummaryItems';
 
 jest.spyOn(utils, 'getComputedReportItems');
 
 const props: ReportSummaryItemsProps = {
-  report: { data: [] },
+  intl: {
+    formatMessage: jest.fn((m, v) => JSON.stringify(v)),
+  } as any,
```

Comment:
Can this be mocked in testEnv.ts?

Reply:
I've been having a tough time with the intl stuff, there's no place in the app that is properly mocking it and i haven't had luck using the code that looks like it was attempting to mock it before.  How could I use testEnv.ts for this?

Reply:
I meant `_mocks_/react-intl.tsx`. We have mocked some `intl` there, but may need to be adjusted